File: app/src/main/java/com/ghstudios/android/features/armorsetbuilder/ASBSetListActivity.java
package com.ghstudios.android.features.armorsetbuilder;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import com.ghstudios.android.mhgendatabase.R;
import com.ghstudios.android.GenericActivity;
import com.ghstudios.android.MenuSection;

public class ASBSetListActivity extends GenericActivity {


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle(R.string.activity_asb_sets);

        // Tag as top level activity
        super.setAsTopLevel();
    }

    @Override
    protected int getSelectedSection() {
        return MenuSection.ARMOR_SET_BUILDER;
    }

    @Override
    protected Fragment createFragment() {
        return new ASBSetListFragment();
    }
}

File: app/src/main/java/com/ghstudios/android/features/armorsetbuilder/list/ASBSetListActivity.kt
package com.ghstudios.android.features.armorsetbuilder.list

import com.ghstudios.android.BasePagerActivity
import com.ghstudios.android.mhgendatabase.R
import com.ghstudios.android.MenuSection
import com.ghstudios.android.features.armorsetbuilder.talismans.ASBTalismanListFragment

/**
 * Main activity for the Armor set list
 */
class ASBSetListActivity : BasePagerActivity() {

    override fun onAddTabs(tabs: TabAdder) {
        setTitle(R.string.activity_asb_sets)

        // Tag as top level activity
        super.setAsTopLevel()

        tabs.addTab(R.string.activity_asb_sets) { ASBSetListFragment() }
        tabs.addTab(R.string.asb_dialog_talisman_title) { ASBTalismanListFragment() }
    }

    override fun getSelectedSection(): Int {
        return MenuSection.ARMOR_SET_BUILDER
    }
}
